js/scheduler.js

```diff
@@ -199,35 +199,17 @@
 			this.toggleState('enable');
 		},
 
-		setUtcTime: function setUtcTime(d, t, offset) {
-			var date = d.split('-');
-			var time = t.split(':');
+		setUtcTime: function setUtcTime(day, time, offset) {
+			var dateArray = day.split('-');
+			var timeArray = time.split(':');
 			function z(n) {
 				return (n < 10 ? '0' : '') + n;
 			}
 
-			var utcDate = new Date(Date.UTC(date[0], (date[1] - 1), date[2], time[0], time[1], (time[2] ? time[2] : 0)));
+			var utcDate = new Date(Date.UTC(dateArray[0], (dateArray[1] - 1), dateArray[2], timeArray[0], timeArray[1], (timeArray[2] ? timeArray[2] : 0)));
 
 			if (offset === 'Z') {
 				utcDate.setUTCHours(utcDate.getUTCHours() + 0);
-			} else {
```

Comment:
I've only confirmed that it does work with it out and doesn't with it. I'm not sure why it is present. It's supposed to add an offset based on the timezone, but it's like it adds a second offset, or it could be this was designed to work without timezones specified in the input